frontend_tests/node_tests/poll_widget.js

```diff
@@ -38,6 +38,7 @@ run_test("PollData my question", () => {
 
     const data_holder = new PollData({
         current_user_id: me.user_id,
+        message_sender_id: me.user_id,
         is_my_poll,
```

Comment:
Can we stop passing this in, and do `message_sender_id === current_user_id` when we need this instead?

Reply:
There are three relevant user_ids:
* the original message sender
* me
* the person sending the event

On the UI side, I need to know if (me === original message sender), so that I can make the question editable.

On the inbound side, I need to know if (them === original message sender), so I can trust their message.

For the latter, my original approach was to pass in the original message sender with every event, but that causes a lot of stuff to have to be plumbed down.  Since the message sender never changes, it makes sense to just send it in when you activate the widget.